src/pages/sponsors.js

```diff
@@ -0,0 +1,107 @@
+import React from 'react'
+import { Helmet } from "react-helmet"
+import styles from "../styles/sponsors.module.css"
+import { graphql } from 'gatsby'
+
+const SponsorsPage = ( {data} ) => {
+
+	const getImageUrl = (companyName) => {
+		const URLRegex = /((([A-Za-z]{3,9}:(?:\/\/)?)(?:[\-;:&=\+\$,\w]+@)?[A-Za-z0-9\.\-]+|(?:www\.|[\-;:&=\+\$,\w]+@)[A-Za-z0-9\.\-]+)((?:\/[\+~%\/\.\w\-_]*)?\??(?:[\-\+=&;%@\.\w_]*)#?(?:[\.\!\/\\\w]*))?)/ ;
+		return URLRegex.test(companyName) ? companyName : 'https://res.cloudinary.com/chicagojs/image/upload/2018Sponsors/' + companyName;
+	}
+
+	const grabTieredSponsors = (tier) => {
+		const sponsorsArray = data.allSponsorsDataJson.edges.find(data => data.node.title === tier);
```

Comment:
`sponsors` maybe? instead of sponsorsArray

Reply:
is casing going to be an issue with title -> tier?

Reply:
now that I look more into the code - might be nice to make an enum for tiers so you can do something like `Tiers.GOLD` and then casing and such can be very consistent. That would make my above comment moot

Reply:
ya i like this idea